daffodil-core/src/main/scala/org/apache/daffodil/core/dsom/SchemaComponent.scala

```diff
@@ -330,19 +330,31 @@ final class Schema private (
   }
 
   /**
-   * Given a name, retrieve the appropriate object.
+   * Given a qname, retrieve the appropriate object.
    *
    * This just scans each schema document in the schema, checking each one.
    */
-  def getGlobalElementDecl(name: String) = {
+  def getGlobalElementDecl(qname: RefQName): Option[GlobalElementDecl] = {
```

Comment:
This function now is slightly differently than the other `getFoo` functions below. That always gives me a pause when a bunch of things look like they should be basically the same but have subtle differences. Why does this one need to be different? Why is it okay for those to still lookup based on name, but this one needs a qname?